packages/phrases/src/locales/ko/errors.ts

```diff
@@ -41,22 +41,22 @@ const errors = {
     identity_not_exists: '소셜 계정이 아직 등록되지 않았어요.',
     identity_exists: '소셜 계정이 이미 등록되있어요.',
     invalid_role_names: '직책 명({{roleNames}})이 유효하지 않아요.',
-    cannot_delete_self: 'You cannot delete yourself.', // UNTRANSLATED
-    sign_up_method_not_enabled: 'This sign up method is not enabled.', // UNTRANSLATED
-    sign_in_method_not_enabled: 'This sign in method is not enabled.', // UNTRANSLATED
-    same_password: 'New password cannot be the same as your old password.', // UNTRANSLATED
-    require_password: 'You need to set a password before signing-in.', // UNTRANSLATED
-    password_exists: 'Your password has been set.', // UNTRANSLATED
-    require_username: 'You need to set a username before signing-in.', // UNTRANSLATED
-    username_exists: 'This username is already in use.', // UNTRANSLATED
-    require_email: 'You need to add an email address before signing-in.', // UNTRANSLATED
-    email_exists: 'This email is associated with an existing account.', // UNTRANSLATED
-    require_sms: 'You need to add a phone number before signing-in.', // UNTRANSLATED
-    sms_exists: 'This phone number is associated with an existing account.', // UNTRANSLATED
-    require_email_or_sms: 'You need to add an email address or phone number before signing-in.', // UNTRANSLATED
-    suspended: 'This account is suspended.', // UNTRANSLATED
-    user_not_exist: 'User with {{ identity }} has not been registered yet', // UNTRANSLATED,
-    missing_profile: 'You need to provide additional info before signing-in.', // UNTRANSLATED
+    cannot_delete_self: '자기 자신을 삭제할 수 없어요.',
+    sign_up_method_not_enabled: '이 회원가입 방법은 활성화 되어있지 않아요.',
+    sign_in_method_not_enabled: '이 로그인 방법은 활성화 되어있지 않아요.',
+    same_password: '새로운 비밀번호는 이전 비밀번호와 같으면 안되요.',
+    require_password: '로그인 전에 비밀번호를 설정해야해요.',
+    password_exists: '비밀번호가 설정되었어요.',
+    require_username: '로그인 전에 사용자명을 설정해야해요.',
+    username_exists: '이 사용자명은 이미 사용 중이에요.',
+    require_email: '로그인 전에 이메일 주소을 설정해야해요.',
+    email_exists: '이 이메일 주소은 다른 계정에 이미 사용 중이에요.',
+    require_sms: '로그인 전에 휴대전화번호를 설정해야해요.',
+    sms_exists: '이 번호는 다른 계정에 이미 사용 중이에요.',
+    require_email_or_sms: '로그인 전에 이메일 주소 또는 휴대전화번호를 설정해야해요.',
+    suspended: '이 계정은 일시 정지되었어요.',
+    user_not_exist: '{{identity}}의 사용자가 아직 등록되지 않았어요.',
```

Comment:
We are going to refine these error messages because it's confusing to even ourselves. Please hold on for a few days until we merge in the latest changes for this part.
For example, the `email_exists_registered` message is trying to tell the user that the provided email has already been taken by other users (so you can't bind it to your account), whereas `email_exists` is for the case that your account has already associated an email address, so you don't need to bind again on sign-in.

Reply:
That makes sense. It looks like, even I made a mistake in that case.

I will wait for that.

Reply:
Hi @pemassi, the refactor is done in #2598
Please pull the latest code and rebase your PR from master. Thank you so much for the continuous support!